### PR TITLE
zh-Hans: Translation for basic-syntax.md

### PR DESCRIPTION
**Pull request by sylxjtu**
Part of #324.

### CODE REVIEW

**Comment by mgeisler** [comment]
@sylxjtu, added the issue number above -- that way the PR is linked back to the issue automatically.

**Review by wnghl — COMMENTED** [review]
The translation is great😊! But this sentence “代码块与作用域使用花括号” may not fully convey the meaning of the word "delimited". Perhaps we could use "代码块和作用域都是由花括号来界定的" or “使用花括号来划分代码块和作用域”？

File: po/zh-Hans.po [hunk 1]
```diff
@@ -2331,6 +2331,12 @@ msgid ""
 "* Keywords like `if` and `while` work the same.\n"
 "* Variable assignment is done with `=`, comparison is done with `==`."
 msgstr ""
+"* 代码块与作用域使用花括号。\n"
```
Comment (mgeisler) [inline po/zh-Hans.po:2334]: Just as a random side comment: @wnghl, let me introduce you to a feature that made my GitHub workflow much faster :smile: GitHub suggestions:

```suggestion
"* 代码块和作用域都是由花括号来界定的。\n"
```

:exploding_head:

Comment (sylxjtu) [reply po/zh-Hans.po]: Thank you for your suggestion, "界定" is the correct word to translate "delimited". I updated translation of the following line with "界定" too.

**Review by wnghl — APPROVED** [review]

**Pull request merged**